README.md

```diff
@@ -27,15 +27,15 @@ modeler made primarily to design real-life objects of any size.
 Parametric modeling allows you to easily modify your design by going back into 
 your model history to change its parameters. 
 
-* **Create 3D from 2D & back** FreeCAD lets you to sketch geometry constrained
+* **Create 3D from 2D and back** FreeCAD lets you to sketch geometry constrained
  2D shapes and use them as a base to build other objects. 
  It contains many components to adjust dimensions or extract design details from 
  3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
  including product design, mechanical engineering and architecture. 
  Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
- you will feel right at home.
+ you will hopefully feel right at home.
```

Comment:
This sentence still rubs me the wrong way, lol.  
I think we can remove line 38 altogether, actually

Reply:
Gone :)